src/scriptdev2/scripts/northrend/borean_tundra.cpp

```diff
@@ -1195,6 +1196,126 @@ CreatureAI* GetAI_npc_seaforium_depth_charge(Creature* pCreature)
+                break;
+            case 4:
+                DoScriptText(SAY_1_MOOTOO_Y, m_creature);
+                break;
+            case 9:
+                if (Player* pPlayer = GetPlayerForEscort())
+                    DoScriptText(SAY_2_MOOTOO_Y, m_creature, pPlayer);
+                break;
+            case 10:
+                m_creature->HandleEmote(EMOTE_ONESHOT_POINT);
+                break;
+            case 12:
+                DoScriptText(SAY_3_MOOTOO_Y, m_creature);
+                break;
+            case 14:
+                DoScriptText(SAY_4_MOOTOO_Y, m_creature);
+                break;
+            case 15:
+                DoScriptText(SAY_5_MOOTOO_Y, m_creature);
+                break;
+            case 16:
+                DoScriptText(SAY_6_MOOTOO_Y, m_creature);
+                break;
+            case 18:
+                DoScriptText(SAY_4_MOOTOO_Y, m_creature);
+                break;
+            case 19:
+                DoScriptText(SAY_7_MOOTOO_Y, m_creature);
+                break;
+            case 20:
+                DoScriptText(SAY_8_MOOTOO_Y, m_creature);
+                break;
+            case 22:
+                DoScriptText(SAY_CREDIT_MOOTOO_Y, m_creature);
+                SetRun();
+                break;
+            case 23:
+                if (Player* pPlayer = GetPlayerForEscort())
+                    pPlayer->GroupEventHappens(QUEST_ESCAPING_THE_MIST, m_creature);
+                if (Creature* pFather = GetClosestCreatureWithEntry(m_creature, NPC_ELDER_MOOTOO, 30.0f))
+                    DoScriptText(SAY_1_ELDER_MOOTOO, pFather);
+                break;
+            case 24:
+                DoScriptText(SAY_9_MOOTOO_Y, m_creature);
+                if (Player* pPlayer = GetPlayerForEscort())
+                if (Creature* pFather = GetClosestCreatureWithEntry(m_creature, NPC_ELDER_MOOTOO, 30.0f))
+                    DoScriptText(SAY_2_ELDER_MOOTOO, pFather, pPlayer);
+                break;
+            case 25:
+                if (Player* pPlayer = GetPlayerForEscort())
+                if (Creature* pFather = GetClosestCreatureWithEntry(m_creature, NPC_ELDER_MOOTOO, 30.0f))
+                    pFather->HandleEmote(EMOTE_ONESHOT_BOW);
+                SetEscortPaused(true);
+                m_creature->ForcedDespawn(10000);
+                break;
+        }
+    }
+
+    void ReceiveAIEvent(AIEventType eventType, Creature* /*pSender*/, Unit* pInvoker, uint32 uiMiscValue) override
+    {
+        if (eventType == AI_EVENT_START_ESCORT && pInvoker->GetTypeId() == TYPEID_PLAYER)
+            Start(false, (Player*)pInvoker, GetQuestTemplateStore(uiMiscValue));
+    }
+};
+
+bool QuestAccept_npc_mootoo_the_younger(Player* pPlayer, Creature* pCreature, const Quest* pQuest)
+{
+    if (pQuest->GetQuestId() == QUEST_ESCAPING_THE_MIST)
+        pCreature->AI()->SendAIEvent(AI_EVENT_START_ESCORT, pPlayer, pCreature, pQuest->GetQuestId());
+    return true;
```

Comment:
Include ```return true;``` in the if statement and surround by { }.
And add a ``` return false;``` statement at the end.

Reason for this, is the following:
If the NPC has more quests, if you always return true in SD2, you won't be able to use dbscripts_on_quest_start / end for any other quest.